CHANGELOG.md

```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2024-05-29
+
+### Added
+
+- Add support to repack a variant ([#214], [#211], [#217], [#219], [#221], [#222], [#228], [#231], [#235], [#236], [#243])
```

Comment:
#236 is a tangential, but standalone feature. Let's add a new line for "Add support to fetch a variant"